data/faq.html

```diff
@@ -893,12 +1071,17 @@ <h3 lang="es">Navegación, rutas e itinerarios</h3>
 mapa. También puedes <a href="http://gpx2kml.com">convertir de GPX a KML</a> o KMZ e 
 <a href="https://support.maps.me/hc/en-us/articles/207895029-How-can-I-import-bookmarks-">importar
 itinerarios y marcadores</a> en formato KML o KMZ.</dd>
+<dd lang="fr">Pas encore. Jusqu'à présent, vous pouvez utiliser l'option "itinéraire récent" (préférences de app - Plan - itinéraire récent).
+Il vous permet d'enregistrer le chemin parcouru pendant une certaine période et les visualiser sur la carte. Vous pouvez aussi
+<a href="http://gpx2kml.com">convertir un GPX en KML</a> ou KMZ et <a href="https://support.maps.me/hc/en-us/articles/207895029-How-can-I-import-bookmarks-">import
+traces et signets</a> en format KML ou KMZ.</dd>
 
 <dt lang="en">Can I use my bookmarks/routes from other apps (like, Google Maps)?</dt>
 <dt lang="ru">Могу ли я воспользоваться своими метками/маршрутами из других приложений 
 (например, Google Maps)?</dt>
 <dt lang="de">Kann ich meine Lesezeichen/Routen von Google Maps verwenden?</dt>
 <dt lang="es">¿Puedo utilizar mis marcadores o rutas de Google Maps?</dt>
+<dt lang="fr">Puis-je utiliser des repères et itinéraire en provenance d'autres applications (telle que google Maps)?</dt>
```

Comment:
Again, no space before the `?`. Or there are extra spaces near the beginning of this document.